sdk/data_cosmos/src/headers/common_headers.rs

```diff
@@ -0,0 +1,49 @@
+use azure_core::{
+    headers::{self, Headers},
+    SessionToken,
+};
+use time::OffsetDateTime;
+
+use crate::ResourceQuota;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct CommonHeaders {
```

Comment:
Can we add documentation about what this is and when it should be used here? Seeing the context of the PR helps me make sense of it, but I'm not sure if I would quite understand what it does if I saw this just on its own.